giotto/diagrams/tests/test_relevant_holes.py

```diff
@@ -0,0 +1,39 @@
+"""Testing for ReleventHoles"""
+
+import numpy as np
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from giotto.diagrams import RelevantHoles
+
+X_rh = np.array([[[0, 1, 0], [0, 4, 0], [2, 3, 1], [0, 0, 2]],
+                 [[0, 3, 0], [0, 2, 0], [1, 2, 1], [0, 0, 2]]])
+
+
+def test_rh_not_fitted():
```

Comment:
I would prefer to have explicit test names so I don't have to guess what "rh" is when a test fails. For example, here you could write something like `test_throw_exception_when_not_fitted`. The advantage of this formulation is that it remains valid even if we rename the class `RelevantHoles()`. A similar logic applies to the other test names